### PR TITLE
appfolio_update2_updated

### PR DESCRIPTION
**Pull request by Pinkaban**

### CODE REVIEW

File: ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md [hunk 1]
```diff
@@ -1,38 +1,52 @@
...
 
-* In regards to data access, we have finally gotten access to the Appfolio dataset, which is composed of:
-  * images of the invoices
-  * csv files composed of bounding box annotations for each word in the invoices
-  * the metadata of each invoice upload, which includes a Total amount value that we can use as validation for our models that extract the total amount from an invoice
+![FUNSDpred](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/FUNSDpred.png?raw=true)
```
Comment (syoh) [inline ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md]: For the image `FUNSDpred.png` in the same directory as the markdown file, you can use `some-text` 
Refer to https://jupyterbook.org/content/figures.html for more info

File: ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md [hunk 2]
```diff
@@ -1,38 +1,52 @@
...
 
-* Worked on combining datasets and cleaning them so as to get them in the format we needed for our models.
+* Worked on cleaning datasets so as to get them in the format we needed for our models (as mentioned before within the individual sections). For detectron2, this included creating bounding boxes around every text of interest and then including the width, height, and label for the bounding box. This is then included with the actual features of the image in a json format. 
 * Implemented a function to visualize the predictions of our models. This is useful in order to see what the label for each bounding box is.
 
 
-![InvoiceLabels](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/InvoiceLabels.png?raw=true)
+![Invoice_Label](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/Invoice_Label.PNG?raw=true)
```
Comment (syoh) [inline ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md]: Same here

For the image `FUNSDpred.png` in the same directory as the markdown file, you can use `some-text` 
Refer to https://jupyterbook.org/content/figures.html for more info

**Review by syoh — CHANGES_REQUESTED** [review]

**Review by Pinkaban — COMMENTED** [review]
I have made the changes suggested by @syoh. Thank you professor for the suggestions!

**Review by syoh — APPROVED** [review]
looks good

**Pull request merged**